### PR TITLE
Paris

### PR DESCRIPTION
**Pull request by parriiss**

### CODE REVIEW

File: src/Controller/Controller.js [hunk 1]
```diff
@@ -211,18 +223,133 @@ function update_filename_at_DB(db, padId, newName) {
...
+/**
+ * Delete a file from DB,locally and the map
+ * 200(OK) / 404(Pad Not Found) / 500(Internal Server Error)
+ * @param {http request} req 
+ * @param {http request} res 
+ */function DeleteFile(req, res) {
+	var pad_obj = PadMap.get(req.body.id);
+	if (pad_obj === undefined) {
+		console.log("Pad not found");
+		res.status(404).send();
```
Comment (ManosKokos) [inline src/Controller/Controller.js]: the res.status(..).send(); had weird behaviour when I used this? Is this tested, if not check it out

File: src/Controller/Controller.js [hunk 2]
```diff
@@ -211,18 +223,133 @@ function update_filename_at_DB(db, padId, newName) {
...
+		return;
+	}
+	fs.unlink(originalPath, (err) => {
+		if (err) {
+			fs.rename(recPath, originalPath, (err) => {
+				if (err) {
+					res.status(500).send();
+					return;
+				}
+				console.log('renamed complete');
```
Comment (ManosKokos) [inline src/Controller/Controller.js]: renamed complete :rofl:

File: src/Controller/Controller.js [hunk 3]
```diff
@@ -211,18 +223,133 @@ function update_filename_at_DB(db, padId, newName) {
...
+		console.log('successfully deleted ' + originalPath);
+	});
+	var db = Database_Connect();
+	if (db === null) {
+		fs.rename(recPath, originalPath, (err) => {
+			if (err) {
+				res.status(500).send();
+				return;
+			}
+			console.log('renamed complete');
```
Comment (ManosKokos) [inline src/Controller/Controller.js]: :rofl:

File: src/index.js [hunk 4]
```diff
@@ -22,7 +22,8 @@ app.put('/Edit', control.Edit);
 
 app.post('/CreateNewPad', control.NewPad);
 app.post('/RenameFile', control.RenamePad);
-
+app.delete('/DeleteFile',control.DeletePad);
+app.post('/EmptyPad',control.EmptyPad);
```
Comment (ManosKokos) [inline src/index.js]: in RESTfull API shouldn't we avoid verbs?? We have completely missed this...

File: src/Controller/Controller.js (hunk 1 above)
Comment (Deletefile) [reply src/Controller/Controller.js]: You accidentaly tagged me here ahahah

**Pull request merged**